learn/dapps-list/dapp-directory.md

```diff
@@ -0,0 +1,299 @@
+
+Despite the distinction between the Moonbeam and Moonriver DApp directories, the submission process remains the same. To list your project on the DApp Directory, you must submit a pull request to the [Moonbeam Foundation's App Directory Data repository on GitHub](https://github.com/moonbeam-foundation/app-directory-data){target=\_blank}. This guide outlines the necessary data and formatting specifics for your submission.
+
+![The Moonbeam DApp Directory home page](/images/learn/dapps-list/dapp-directory/directory-1.webp)
+
+## Overview of the Project Data {: #overview-project-data }
+
+There are four main sources of data that are used for a project's listing in the Moonbeam DApp Directory:
+
+- **Core Project Data** - core project data such as descriptions, logos, and screenshots. This data also includes IDs used to query data from external platforms
+- **Active Users and Transaction Volume** - on-chain data based on smart contract activity for all of the contracts associated with the project. Data is discovered via the use of contract labeling in Moonscan, which is then indexed by Web3Go and consumed by the DApp Directory
+- **TVL Data** - TVL data for the protocol, sourced from the project's listing on DefiLlama
+- **Project Token Information** - token information, which is sourced from the project's listing on CoinGecko
+
+## Prerequisites for Using External Data Sources {: #configuring-external-data-sources }
+
+Before pulling data from the mentioned sources, certain prerequisites must be fulfilled. However, it's worth noting that these steps may not apply to all project types. For instance, in the case of wallets, where there are no smart contracts, the DApp Directory is currently unable to display user and transaction activity data.
+
+### Configure the Data Source for Active Users and Transaction Volume {: #configure-active-users }
+
+For projects that have smart contracts deployed on Moonbeam or Moonriver, it is important that those contracts can be linked to the DApp Directory project data.
+
+The end-to-end flow for linking smart contract activity to the DApp Directory is as follows:
+
+1. The smart contract owner fills in the form to label contracts on Moonscan
+2. The contracts become labeled in Moonscan
+3. Periodically, the entire list of labeled contracts is exported and transmitted to Web3Go to be ingested
+4. Every hour, Web3Go loads smart contract activity within Moonbeam and Moonriver and runs a job to index this data by the labels
+
+To get your project's smart contracts properly labeled on [Moonscan](https://moonscan.io){target=\_blank}, please refer to Web3Go's documentation on the [Labeling Structure](https://web3go.gitbook.io/moonscan-smart-contract-label-for-projects/labeling-structure){target=\_blank} and [How to Submit Contract Information](https://web3go.gitbook.io/moonscan-smart-contract-label-for-projects/how-to-submit-contract-information){target=\_blank} on Moonscan.
```

Comment:
Ah I see this is great. So maybe just include the _How to Submit Contract Information_ link as a link in Step 1 as well.